test/export/later.js

```diff
@@ -4,15 +4,18 @@ export default "default-1";
 export let val = "value-1";
 
 export function exportAgain() {
-  module.export("default", exports.default = "default-2");
+  // Neither of these re-export styles should work, because the original
+  // export default still takes precedence over anything else.
+  module.exportDefault(exports.default = "default-2");
+
+  // This style also does not work, because the getter function for the
+  // variable val is all that matters.
+  exports.val = "ignored";
```

Comment:
@jdalton Here's proof that tampering with `exports` directly does not interfere with `module.export` getters.